### DISCUSSION TITLE
It's possible to use vanilla mask only?

### DISCUSSION THREAD

**Discussion by douglasjunior** [uNmAnNeR/imaskjs#753] [Q&A] [answered]
**What problem you are trying to solve?**

I would like to know if it is possible to use the mask functions with JavaScript only, without DOM dependency.

The goal is to use mask formatting with Node or React Native, in places that are not just inputs.

**Describe the solution you'd like**

Usage suggestion:

```js
const masker = IMask({
    mask: '+{00} (00) {0} 0000-0000'
  });

const masked = masker.format('+5544999594455');

console.log(masked);
// +55 44 9 9959-4455
```

**Describe alternatives you've considered**

In the past I've used [Vanilla Masker](https://github.com/vanilla-masker/vanilla-masker), but the library has been abandoned and doesn't support TypeScript.

**Comment by uNmAnNeR** [comment, marked as answer]
hey! you can use pipes for it https://imask.js.org/guide.html#pipe

**Reply by douglasjunior** [reply]
Very nice, thank you!
